lib/menu.rb

```diff
@@ -1,12 +1,14 @@
 class Menu
   attr_reader :list
-  
+
   def initialize(list)
   @list = list
   
```

Comment:
I find rubocop -x is great for checking code layout (would it have said this was an extra line?)